src/Api.hs

```diff
@@ -29,12 +29,12 @@ userApp cfg = serve (Proxy :: Proxy UserAPI) (appToServer cfg)
 appToServer :: Config -> Server UserAPI
 appToServer cfg = enter (convertApp cfg) userServer
 
--- | This function converts our 'App' monad into the @ExceptT ServantErr
--- IO@ monad that Servant's 'enter' function needs in order to run the
+-- | This function converts our 'AppT m' monad into the @ExceptT ServantErr
```

Comment:
Haddock syntax is a little weird. Single quoted identifiers are linked, but IIRC only if it's a single word. The right way to write this is `@'AppT' m@` -- `@` denotes a code block, and `'` denotes linking.

Reply:
That's good to know, thank you.